src/Makefile.am

```diff
@@ -243,6 +243,18 @@ libbitcoin_wallet_a_SOURCES = \
   $(BITCOIN_CORE_H) \
   $(LIBZCASH_H)
 
+TROMPEQUIHASH_SOURCES = \
```

Comment:
I'd prefer to use `src/equihash/SOLVERNAME` for imported solver code, so this code would be located in `src/equihash/tromp`.